sycl/doc/GetStartedGuide.md

```diff
@@ -517,7 +518,7 @@ class CUDASelector : public cl::sycl::device_selector {
       const std::string DeviceName = Device.get_info<device::name>();
       const std::string DeviceVendor = Device.get_info<device::vendor>();
 
-      if (Device.is_gpu() && (DeviceName.find("NVIDIA") != std::string::npos)) {
+      if (Device.is_gpu() && (DeviceName.find("CUDA") != std::string::npos)) {
```

Comment:
How it this change related to the documentation update?
BTW. related change: https://github.com/intel/llvm/pull/2066/files

Reply:
I was updating the getting started guide as part of the documentation. Regardless,re-thinking it after seeing !2066, I think the most accurate way of selecting the CUDA device is actually the one we use on the examples https://github.com/codeplaysoftware/SYCL-For-CUDA-Examples/blob/5d257b29d0cd25017c3ff680976b615eb8673d34/example-02/sycl_sgemm.cpp#L32-L34 

```cpp
    const std::string DriverVersion = Device.get_info<device::driver_version>();

    if (Device.is_gpu() && (DriverVersion.find("CUDA") != std::string::npos)) {
      std::cout << " CUDA device found " << std::endl;
      return 1;
    };
    return -1;
```

I've updated the PR accordingly